VirtoCommerce.Storefront/Domain/Catalog/CatalogWorkContextBuilderExtensions.cs

```diff
@@ -18,9 +18,9 @@ public static Task WithCatalogsAsync(this IWorkContextBuilder builder)
             //Initialize catalog search criteria
             var productSearchcriteria = new ProductSearchCriteria(workContext.CurrentLanguage, workContext.CurrentCurrency, workContext.QueryString)
             {
-                UserGroups = workContext.CurrentUser?.Contact?.UserGroups ?? new List<string>(),
-                SortBy = "priority-descending;name-ascending"
+                UserGroups = workContext.CurrentUser?.Contact?.UserGroups ?? new List<string>()                               
             };
+            productSearchcriteria.SortBy = productSearchcriteria.SortBy ?? "priority-descending;name-ascending";
```

Comment:
I think better to check using string.IsNullOrEmpty(), as "" value should be treat as default sorting.